### DISCUSSION TITLE
Intro Bug - Can't use attack or use/roll

### DISCUSSION THREAD

**Discussion by magicienlord** [leoetlino/project-restoration#243] [General]
Hey guys sorry to bother you, I doubt this thread will even be alive but I'm playing the Project Restoration on Citra and I'm stuck at the intro after you're changed into a Deku. I can't seem to attack (spin dance) or open the door, the buttons just won't work for me. Has anyone had the same problem?

**Comment by leoetlino** [comment]
That's not a Project Restoration problem, that's just the controller being configured incorrectly in Citra and there's nothing this project can do. You have to fix your controller configuration in Citra.
